website/src/components/Messages/MessageTableEntry.tsx

```diff
@@ -111,27 +107,28 @@ export function MessageTableEntry({ message, enabled, highlight, showAuthorBadge
   const { t } = useTranslation(["message"]);
 
   return (
-    <HStack w={["full", "full", "full", "fit-content"]} gap={0.5} alignItems="start">
+    <HStack w={["full", "full", "full", "fit-content"]} gap={0.5} alignItems="start" maxW="full" position="relative">
```

Comment:
Have to move position relative to this element to fix         
![image](https://user-images.githubusercontent.com/33456881/219877296-6f0a1b30-6198-4079-8a70-d95af9dc4108.png)
Not sure why adding overflow-x: auto makes the badge hidden.